snapcraft/internal/build_providers/_multipass/_multipass_command.py

```diff
@@ -26,9 +26,9 @@
 logger = logging.getLogger(__name__)
 
 
-def _run(command: Sequence[str]) -> None:
+def _run(command: Sequence[str], stdin=None) -> None:
```

Comment:
I'd make this default to `subprocess.DEVNULL`, if the intent really is to not use it where possible. Then when you want the default behaviour you do `stdin=None` (as you are already below). The diff should shrink quite a bit, which is a bonus.

Reply:
sounds like a good idea